File: tests/javascript/third_parties/open_telemetry/testdata/datatype_in_record_exception.js
import * as opentelemetry from "@opentelemetry/sdk-node"

var span = opentelemetry.trace.getSpan()

try {
  // something
} catch (err) {
  span.recordException(err)
// bearer:expected javascript_third_parties_open_telemetry
  span.recordException(currentUser.ipAddress)
}

File: tests/ruby/third_parties/open_telemetry/testdata/datatype_in_record_exception.rb
# set status
current_span = OpenTelemetry::Trace.current_span

begin
  1/0 # something that obviously fails
rescue
  1+1
  # bearer:expected ruby_third_parties_open_telemetry
  current_span.status = OpenTelemetry::Trace::Status.error("error for user #{current_user.email}")
end

# record exception to span (recommended together with set status)
current_span = OpenTelemetry::Trace.current_span
begin
  1/0 # something that obviously fails
rescue Exception => ex
  current_span.status = OpenTelemetry::Trace::Status.error("error message here!")
  current_span.record_exception(ex)
  # bearer:expected ruby_third_parties_open_telemetry
  current_span.record_exception(ex, attributes: { "user.ip" => user.ip_address })
end